test/testrunner.py

```diff
@@ -449,9 +449,14 @@ def main():
             sys.exit(pytest.main(pytest_cmd))
 
         else:
-            run_sagemaker_remote_tests(
-                [image for image in standard_images_list if not ("tensorflow-inference" in image and "py2" in image)]
-            )
+            if "neuron" in dlc_images:
+                run_sagemaker_remote_tests(
+                    [image for image in standard_images_list if ("mxnet" in image)]
+                )
```

Comment:
Just noticed that the sm test jobs for PyTorch and TF fail because there is no empty report file generated by `sm_utils.gnerate_empty_report()`. If the image isn't an MXNet image, then we will need to generate an empty report.